Traceback (most recent call last):
  File "/Users/johnmyers/opt/anaconda3/envs/python-2023-08-v2/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/johnmyers/opt/anaconda3/envs/python-2023-08-v2/lib/python3.11/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/johnmyers/opt/anaconda3/envs/python-2023-08-v2/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/johnmyers/opt/anaconda3/envs/python-2023-08-v2/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Users/johnmyers/opt/anaconda3/envs/python-2023-08-v2/lib/python3.11/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/Users/johnmyers/opt/anaconda3/envs/python-2023-08-v2/lib/python3.11/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/johnmyers/opt/anaconda3/envs/python-2023-08-v2/lib/python3.11/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
sns.kdeplot(x=srs_sales, bw_method=0.3)
plt.gcf().set_size_inches(w=8, h=2)
plt.xlabel('quantity')
plt.ylabel('probability density')
plt.tight_layout()
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[19], line 1[0m
[0;32m----> 1[0m sns[38;5;241m.[39mkdeplot(x[38;5;241m=[39m[43msrs_sales[49m, bw_method[38;5;241m=[39m[38;5;241m0.3[39m)
[1;32m      2[0m plt[38;5;241m.[39mgcf()[38;5;241m.[39mset_size_inches(w[38;5;241m=[39m[38;5;241m8[39m, h[38;5;241m=[39m[38;5;241m2[39m)
[1;32m      3[0m plt[38;5;241m.[39mxlabel([38;5;124m'[39m[38;5;124mquantity[39m[38;5;124m'[39m)

[0;31mNameError[0m: name 'srs_sales' is not defined

